cli/cli.go

```diff
@@ -308,6 +313,26 @@ func SetupCLI(args []string) error {
 							CopySnapshot(ctx, os.Stdout)
 					},
 				},
+				{
+					Name:        "ssh",
+					Description: snapshotSSHDescription,
+					Usage:       "Write rootfs to ssh host",
+					UsageText: "mender snapshot ssh " +
+						"[command options] -- " +
+						"<ssh options>",
+					Action: func(ctx *cli.Context) error {
+						return runOptions.DumpSSH(ctx)
+					},
+					Flags: []cli.Flag{
+						cli.StringFlag{
+							Name: "file, f",
```

Comment:
I propose using `--outfile` instead of `--file`, since we already use the `--file` argument in mender-artifact to mean input file. Less confusing.

`-f` is actually a valid ssh option, but I think that's ok, since it's unlikely to be used in this context, and it's still possible to specify after the `--`. Switching to `-o` is tempting, but we should probably not, because `-o` is a quite relevant argument to use (for example to specify `-o StrictHostKeyChecking=false`).

So in other words, my proposal is `"outfile, f"`. Makes sense?

Reply:
Hmm... Or `"out-file, o"`?
It's a bit awkward, but you have to force ssh arguments to be positional (i.e. after `--`) otherwise the cli package caches it and throws an unrecognized option error.

Reply:
Aha! Yes, agree, if they have to be positional anyway, then we should definitely use `-o`.